c10/macros/Macros.h

```diff
@@ -409,10 +409,10 @@ __host__ __device__
 #define C10_HOST_CONSTEXPR_VAR constexpr
 #endif
 
-#if defined(__CUDA_ARCH__) && defined(CUDA_VERSION) && CUDA_VERSION <= 9200
-#define C10_CONSTEXPR_EXCEPT_CUDA92
+#if defined(__CUDA_ARCH__) && (__CUDA_ARCH__ <= 9200)
```

Comment:
Was the problem that CUDA_VERSION wasn't defined in CUDA 9.2?

Reply:
I assume problem is codegen. I did take a look of the generated cmake file. Even though `CUDA_VERSION` is set, it is not passed to nvcc compilation command.
The macro __CUDA_ARCH__ is defined as long as we are using nvcc.